tests/test_mouse.py

```diff
@@ -0,0 +1,193 @@
+    x, y, _, _ = await evaluate_dimensions(page)
+    await page.mouse.move(x + 2, y + 2)
+    await page.mouse.down()
+    await page.mouse.move(100,100)
+    await page.mouse.up()
+    result = await page.evaluate("""
+        () => {
+            const textarea = document.querySelector('textarea');
+            return textarea.value.substring(textarea.selectionStart, textarea.selectionEnd);
+    }""")
+    assert result == text
+
+
+@sync
+@chrome_only
+async def test_trigger_hover_state(isolated_page, server):
+    """should trigger hover state"""
+    page = isolated_page
+    await page.goto(server / 'input/scrollable.html')
+    await page.hover('#button-6')
+    button6_id = await page.evaluate("() => document.querySelector('button:hover').id")
+    assert button6_id == 'button-6'
+    await page.hover('#button-2')
+    button2_id = await page.evaluate("() => document.querySelector('button:hover').id")
+    assert button2_id == 'button-2'
+    await page.hover('#button-91')
+    button91_id = await page.evaluate("() => document.querySelector('button:hover').id")
+    assert button91_id == 'button-91'
+
+
+@sync
+@chrome_only
+async def test_trigger_hover_removed_window_node(isolated_page, server):
+    """should trigger hover state with removed window.Node"""
+    page = isolated_page
+    await page.goto(server / 'input/scrollable.html')
+    await page.evaluate("() => delete window.Node")
+    await page.hover('#button-6')
+    assert await page.evaluate("() => document.querySelector('button:hover').id") == 'button-6'
+
+
+@sync
+async def test_set_modifier_keys_onclick(isolated_page, server, firefox):
+    """should set modifier keys on click"""
+    page = isolated_page
+    await page.goto(server / 'input/scrollable.html')
+    await page.evaluate("""
+        () => {
+            const btn = document.querySelector('#button-3');
+            btn.addEventListener('mousedown', e => window.lastEvent = e, true);
+        }
+    """)
+    modifiers = {'Shift': 'shiftKey', 'Control': 'ctrlKey', 'Alt': 'altKey', 'Meta': 'metaKey'}
+    # In Firefox, the Meta modifier only exists on Mac
+    if firefox and platform.system() != 'darwin':
+        del modifiers['Meta']
+    for modifier_key, modifier_value in modifiers.items():
+        await page.keyboard.down(modifier_key)
+        await page.click('#button-3')
+        eval_modifier = await page.evaluate("mod => window.lastEvent[mod]", modifier_value)
+        assert eval_modifier, f"{modifier_key, modifier_value} should be true"
+        await page.keyboard.up(modifier_key)
+
+    await page.click('#button-3')
+    for modifier_key, modifier_value in modifiers.items():
+        assert not await page.evaluate("mod => window.lastEvent[mod]", modifier_value),  \
+            f"{modifier_key, modifier_value} should be false"
+
+
+@sync
+@chrome_only
+async def test_tween_mouse_movement(isolated_page):
+    """should tween mouse movement"""
```

Comment:
tween is apparently short for tweenager, I'm pretty sure they meant something like this:
```suggestion
async def test_mousetrigger_fires_between_mouse_movement(isolated_page):
    """mousetrigger should fire between mouse movement"""
```

Reply:
async def test_mouse_fires_events_between_movement(isolated_page):
    """Mouse fires the events with x, y position on mouse movement."""